mantidproject/mantid
# Split Activation Fit function into two seperate functions

**Description of work.**
Split the Activation Fitting function for Muons into two separate functions, one for Kelvin and one for meV.

**To test:**

1. Enable the model analysis tab and plot (see #32572 for details on how to do this)
3. Open Muon Analysis GUI
4. Change Instrument to EMU
5. Load runs 20901-18
6. Go to the Grouping Tab
7. Guess Alpha
8. Go to fitting tab and add a DynamicKuboToyabe Fitting Function
9. Fix the Field to 0, and set Delta to 0.35 and Nu to 0.01
10. Go to the Sequential Fitting Tab and select runs 20901-17 and then sequentially fit selected
11. Go to the Results Tab
12. Select Temp_Sample from the log results and then click Output Results
13. Open the Model fitting tab and change x axis to Temp_sample and y axis to Nu
14. Add an ActivationK Fit Function and fit (leave Unit as K). AttemptRate = 33.4 +- 3.6 us^-1 and Barrier = 589.7+-13.9 and ChiSquared should be about 3.75 or better.

Note:
We have not been provided with data in meV to be able to directly test ActivationmeV fitting function. However as the conversion from Kelvin to meV is provided by https://github.com/mantidproject/mantid/blob/15ef1d68a3427cf52dbf91faa2072ca5c5bae02d/Framework/Kernel/inc/MantidKernel/PhysicalConstants.h#L63 which is already tested and the above tests the other part of the functionality ActivationmeV uses then provided the above functional test works, the code is reviewed and all the CI tests pass then ActivationmeV should be good to go.

Fixes #33090. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
